plan/expr_to_pb.go

```diff
@@ -24,13 +24,14 @@ import (
 	"github.com/pingcap/tipb/go-tipb"
 )
 
-func expressionsToPB(exprs []expression.Expression, client kv.Client) (pbExpr *tipb.Expr, remained []expression.Expression) {
+func expressionsToPB(exprs []expression.Expression, client kv.Client) (pbExpr *tipb.Expr, remained []expression.Expression, pushed []expression.Expression) {
```

Comment:
The first return value `pbExpr` is also pushed, I think move `pushed` before `remained` is better.